exercise_2/Student.hpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by denys on 10.07.19.
+//
+
+#pragma once
+
+
+#include "IPerson.hpp"
+
+class Student : public IPerson {
```

Comment:
Shall also inherit from IRecord, i.e.:

`class Student: public IPerson, public IRecord `

Reply:
My IPerson class inherits from IRecord class. 
As I understand if I have Child class that inherits from IPerson, it will automatically inherit all properties from IRecord.

Reply:
Replied already on slack ;)

The thing is that you should have two separate interfaces (`IPerson` **and** `IRecord`)